scripts/pre_commit_hook.py

```diff
@@ -46,6 +46,15 @@ def _install_hook():
     try:
         os.symlink(os.path.abspath(__file__), pre_commit_file)
         print 'Created symlink in .git/hooks directory'
+
+        chmod_cmd = ['chmod', '-x', pre_commit_file]
```

Comment:
(1) Would this work if the hook is already installed?

(2) Doesn't -x make it unexecutable?

Reply:
(1) Hmm, no. I was thinking from a fresh install perspective, my bad.
(2) Oops, should be +x.

Reply:
Fixed, ptal, thanks!